rel/files/ejabberd.cfg

```diff
@@ -662,18 +662,15 @@
                            % default: 600 seconds
                            % {resume_timeout, 600}
                           ]},
-%  {mod_muc_light, [
-%                   {host, "muclight.@HOST@"}
-%                  ]},
-  {mod_muc, [
```

Comment:
Wouldn't it be simpler to check if these two can peacefully coexist on separate subdomains?